benchmarks/java/src/main/java/com/scalar/tool/SmallBankBench.java

```diff
@@ -0,0 +1,202 @@
+    switch (operation) {
+      case "transact_savings":
+      case "deposit_checking":
+      case "write_check":
+        taskParams.add(Integer.toString(amount));
+        taskParams.add(Integer.toString(account1));
+        break;
+      case "send_payment":
+        taskParams.add(Integer.toString(amount));
+        taskParams.add(Integer.toString(account1));
+        taskParams.add(Integer.toString(account2));
+        break;
+      case "amalgamate":
+        taskParams.add(Integer.toString(account1));
+        taskParams.add(Integer.toString(account2));
+        break;
+      default:
+    }
+
+    return taskParams;
+  }
+
+  @Override
+  public Integer call() throws Exception {
+    long durationMillis = duration * 1000;
+    long rampUpTimeMillis = rampUpTime * 1000;
+
+    AtomicBoolean isRunning = new AtomicBoolean(true);
+    ExecutorService executor = Executors.newFixedThreadPool(numThreads);
+    Random rand = new Random();
+    final long start = System.currentTimeMillis();
+    long from = start;
+    FabricClientService service = new FabricClientService(networkConfig, cryptoConfig, commitWaitPolicy);
+    for (int i = 0; i < numThreads; ++i) {
+      executor.execute(
+          () -> {
+            while (isRunning.get()) {
+              int fromId = rand.nextInt(numAccounts);
+              int toId = rand.nextInt(numAccounts);
+              if (toId == fromId) {
+                toId = (toId + 1) % numAccounts;
+              }
+              List<String> taskParams = generateTask(rand);
+
+              try {
+                long eachStart = System.currentTimeMillis();
+                String operation = taskParams.remove(0);
+                service.executeContract("smallbank", operation,
+                  taskParams.toArray(new String[taskParams.size()]));
+                long eachEnd = System.currentTimeMillis();
+                counter.incrementAndGet();
+                if (System.currentTimeMillis() >= start + rampUpTimeMillis) {
+                  totalCounter.incrementAndGet();
+                  latencyTotal.addAndGet(eachEnd - eachStart);
+                }
+              } catch (ContractException e) {
+                errorCounter.incrementAndGet();
+                // e.printStackTrace();
+              } catch (Exception e) {
+                System.out.println("An error ocurred in a thread.");
+                e.printStackTrace();
+                System.exit(-1);
+              }
+            }
+          });
+    }
+
+    long end = start + rampUpTimeMillis + durationMillis;
+    while (true) {
+      long to = System.currentTimeMillis();
+      if (to >= end) {
+        isRunning.set(false);
+        break;
+      }
+      System.out.println(((double) counter.get() * 1000 / (to - from)) + " rps");
+      counter.set(0);
+      from = System.currentTimeMillis();
+
+      try {
+        Thread.sleep(1000);
+      } catch (InterruptedException e) {
+        Thread.interrupted();
```

Comment:
Sorry, it was my mistake. We should have simply ignored it.
```suggestion
        // ignored
```